Helm/plexanisync/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
I'm not 100 % sure but I think this could also be solved with a CronJob instead of starting a pod that does nothing 99 % of the time.

The entrypoint.sh needs to be adjusted so the docker image can also be run without the sleep loop, like by quitting the script if INTERVAL is <= 0.

Reply:
Yep, you're definitely right, a cronjob is the better solution.

I wasn't sure if the changes needed to support the cronjob would be welcomed, so I was pretty conservative with eh chart.

I've gone ahead and updated the entrypoint script.

The chart defaults to a cronjob, with a user configurable schedule, but you can disable the cronjob, in which case a deployment will be created, and the interval setting used instead.

I inspected the chart put;uts with all of the various setting, as well as linting with lube-lint. I haven't had the opportunity to test the chart since I also need a new container image built.

Can you kick off the workflow for a container image? If you don't want to, I can get something built on my fork. Hopefully take care of that tomorrow

Reply:
The PR workflow doesn't actually push docker images so I adjusted the entrypoint myself in the master branch.

Use the latest master image to test your changes.